nautobot/extras/views.py

```diff
@@ -51,6 +58,7 @@
     get_datasource_contents,
     enqueue_pull_git_repository_and_refresh_data,
 )
+from nautobot.extras import choices
```

Comment:
Perhaps move this up and change it to `from nautobot.extras.choices import JobExecutionType`, since that appears to be the only choices value actually needed here?